scenes/enumerated/problem_10349/problem_10349.py

```diff
@@ -0,0 +1,393 @@
+
+        # Մեկ մաս+10֊ը գտնելու հավասարումը
+        one_part_and_10_value = MathTex("50+10=").next_to(
+            one_part_value, DOWN, buff=.4).align_to(one_part_value, LEFT)
+        sixty = MathTex("60").next_to(one_part_and_10_value, RIGHT, buff=0.15)
+        one_part_and_10_label = sixty.copy()
+        self.play(Write(one_part_and_10_value))
+        self.play(Write(sixty))
+        self.wait()
+
+        one_part_and_10_brace = Brace(VGroup(one_part_copy, segment_10), UP)
+        self.play(
+            ReplacementTransform(VGroup(one_part_copy_label, segment_10_label), one_part_and_10_brace),
+            one_part_and_10_label.animate.next_to(one_part_and_10_brace, UP, buff=0.1),
+            one_part_and_10_label.animate.become(Tex(arshak_one_part_len_string).next_to(one_part_and_10_brace, UP, .05).set_color(RED)),
+            one_part_label.animate.become(Tex(souren_one_part_len_string).next_to(one_part, UP, .05).set_color(BLUE))
+        )
+
+        self.play(FadeOut(x8_one_part), FadeOut(x8_one_part_brace), FadeOut(x8_one_part_brace_label),
+                  FadeOut(x8_one_part_brace), FadeOut(x8_one_part))
+
+        # Վերականգնենք սկզբնական ճանապարհի գրաֆիկը ու տեսնենք վերջնական ինչ ստացվեց
+        road_group.restore().align_to(souren_t, DOWN).scale(0.9)
+        self.play(FadeIn(road_group))
+        self.wait()
+
+        souren_road_length = MathTex(r"5\cdot50=250" + km, color=BLUE).next_to(souren_road, DOWN, buff=.5).scale(0.9)
+        self.play(Write(souren_road_length))
+        self.wait()
+        arshak_road_length = MathTex(r"3\cdot60=180" + km, color=RED).next_to(arshak_road, DOWN, buff=.5).scale(0.9)
+        self.play(Write(arshak_road_length))
+
+        recap_souren_s = MathTex(souren_recap_s_string, color=BLUE).scale(0.7).align_to(souren_t, DL)
+        recap_arshak_s = MathTex(arshak_recap_s_string, color=RED).scale(0.7).align_to(arshak_t, DL)
+        recap_souren_v = MathTex(souren_recap_v_string, color=BLUE).scale(0.7).next_to(
+            recap_souren_s, DL, buff=0.3).align_to(recap_souren_s, LEFT)
+        recap_arshak_v = MathTex(arshak_recap_v_string, color=RED).scale(0.7).next_to(
+            recap_arshak_s, DL, buff=0.3).align_to(recap_arshak_s, LEFT)
+        self.play(
+            AnimationGroup(
+                Write(recap_souren_s), Write(recap_arshak_s), Write(recap_souren_v), Write(recap_arshak_v), lag_ratio=1
+            )
+        )
+
+        self.wait(2)
+
+    def get_boy(self, name, svg_name, edge, color=WHITE):
+        boy_svg = SimpleSVGMobject(svg_name)
+        boy = VGroup(boy_svg, Tex(name).next_to(boy_svg, DOWN).set_color(color))
+        boy.scale(.7).to_edge(edge, buff=1).shift(UP)
+        return boy
+
+    def get_part(self, value, label=None, color=WHITE, **kwargs):
+        seg = Segment(
+            ORIGIN,
+            value * RIGHT,
+            label,
+            stroke_width=6,
+            color=color,
+            **kwargs
+        )
+        return seg
```

Comment:
This function might be moved to other location. Take a look at #78

Reply:
I couldn't find a good location for this one. The best I can think of is creating a local `util.py` but even that's too much for this small function.

Reply:
I don't think it's too much, we'll have similar functions in the future. Let's create it in the `lib/` folder.
Or, we can add this ot the `aramanim.py`